scenario-compiler/src/test/scala/pl/touk/nussknacker/engine/definition/clazz/ClassDefinitionTestUtils.scala

```diff
@@ -0,0 +1,17 @@
+package pl.touk.nussknacker.engine.definition.clazz
+
+import pl.touk.nussknacker.engine.api.process.{ClassExtractionSettings, ExpressionConfig}
+
+object ClassDefinitionTestUtils {
+  val DefaultExtractor = new ClassDefinitionExtractor(ClassExtractionSettings.Default)
+  val DefaultDiscovery = new ClassDefinitionDiscovery(DefaultExtractor)
+
+  def buildDefinitionForClasses(classes: Class[_]*): ClassDefinitionSet = ClassDefinitionSet(
```

Comment:
If you are not too much attached to this naming pattern, I'd rather name these methods starting from `create...`. I'd reserve the `build` prefix for the builder pattern - I mean when we have some `Builder.withFoo.withBar.build`

Reply:
Done